contracts/test/umbra-test.js

```diff
@@ -20,8 +53,15 @@ describe('Umbra', () => {
     receiver2,
     acceptor,
     other,
+    relayer,
   ] = accounts;
 
+  // The ethers wallet that will be used when testing meta tx withdrawals
+  const metaWallet = ethers.Wallet.createRandom();
```

Comment:
Yes, this is the stealth address that receives the funds in the meta tx test cases. It needs to be an Ethers wallet since we need it to sign a meta tx for testing withdrawal.